UnitsNet/QuantityFormatter.cs

```diff
@@ -64,7 +64,7 @@ public static string Format<TUnitType>(IQuantity<TUnitType> quantity, string for
         public static string Format<TUnitType>(IQuantity<TUnitType> quantity, string format, IFormatProvider formatProvider)
             where TUnitType : Enum
         {
-            formatProvider ??= CultureInfo.CurrentUICulture;
+            formatProvider = formatProvider ?? CultureInfo.CurrentUICulture;
```

Comment:
You can keep the fancy formatting, I'm still getting used to it too but I don't see why we shouldn't use newer syntax in C#. It is more terse.

Reply:
It wouldn't compile. I'm guessing it's a C# 8.0 feature - same reason the nullable reference types PR won't compile.

Reply:
As per your comment in the other PR, the c#8 syntax might work now that I upgraded to VS2019 image. Maybe. It seems netstandard2.0 defaults to c# 7.3 though, so might require explicitly configuring language version.

Reply:
True. Can do that another day though

Reply:
Added it back when I was updating based on PR :)